swagger.yml

```diff
@@ -292,7 +293,7 @@ paths:
       tags:
         - "Orders"
       description: >-
-        This endpoint modifies the packages in an order. The length of the new packages object passed in must match the original package count of the order.
+        This endpoint modifies the packages in an order. The number of packages in the new package list must be the same as the original order. The weight of each new package should not exceed the weight limit set for the original order's delivery type. For instance, if the original order is labeled as `PARCEL`, the weights of the new packages must stay within the `PARCEL` limit.
```

Comment:
I feel like we only need the first two sentences here. The rest of the text is specific to weight criteria. Is there another place we can put the rest of the text?

Reply:
I'm a bit hesitant to do that because while it makes sense that since it's relating to weight, it should be specified in WeightOptions of #/definitions/UpdateArrayOfPackages. However, weight options itself is a reference to #/definitions/WeightOptions, which is used for GET/CREATE/UPDATE. Since it's a warning only applicable to this endpoint, this was the only way I knew how to make it only appear for this one endpoint